<launch>

  <node name="velocity_control" pkg="kondo_b3mservo_rosdriver" type="velocity_control.py" output="screen"/>
  <node name="generate_command_autodetect_joy"   pkg="kondo_b3mservo_rosdriver" type="generate_command_autodetect_joy.py" output="screen"/>

  <node name="joy_node"   pkg="joy" type="joy_node">
    <param name="autorepeat_rate" value="100"/>
  </node>

</launch>
